src/components/tutorialEditor/index.tsx

```diff
@@ -135,12 +307,148 @@ const TutEditor: React.FC = (): JSX.Element => {
             <StyledTitle>
               <h3>Step Content</h3>
             </StyledTitle>
+            <StepContentBody>
+              <StepContentList>
+                {content.map((value: ContentBlock, index: number) => {
+                  return (
+                    <ContentBlock
+                      key={index.toString()}
+                      onMouseEnter={(): void => {
+                        setHovering(index);
+                      }}
+                      onMouseLeave={(): void => {
+                        setHovering(-1);
+                      }}
+                    >
```

Comment:
Very clever ngl. Although please make this into a sub-component so you can manage state in a more intuitive way.